src/lib.rs

```diff
@@ -1553,22 +1555,46 @@ impl<
             AtomicKeystoreStorage<'a, L, Meta>,
             Assets,
             Transactions<L>,
+            Accounts<L, ViewerKeyPair>,
+            Accounts<L, FreezerKeyPair>,
```

Comment:
Almost feels like this should just return a `KeystoreModel` or something, rather than a tuple of like a dozen things